bundles/org.openhab.binding.tado/src/main/java/org/openhab/binding/tado/internal/TadoHvacChange.java

```diff
@@ -83,7 +90,7 @@ public TadoHvacChange activeUntilChange() {
         return this;
     }
 
-    public TadoHvacChange activeFor(Integer minutes) {
+    public TadoHvacChange activeForMinutes(@Nullable Integer minutes) {
```

Comment:
Does this method make sense for argument being `null` (as called explicitly by `withOperationMode`)? If needed, does `TerminationType` still need to be `TIMER`? Perhaps it would be cleaner and more readable with a distinct method for resetting/clearing minutes, WDYT?

Reply:
As far as I recall, the reason for the `@Nullable` annotation goes back into the JSON (de-) serializer. The JSON element can either be 'null' or a positive number of minutes, and it is important to differentiate between them.

Reply:
@jlaur wherever possible I have converted 'nullable Integer' to a plain 'int'